### PR TITLE
Update eks-managed-nodes.md

### PR DESCRIPTION
**Pull request by lboix**
Adding a tip for the usecase where you are describing your whole EKS cluster in just one single file

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: userdocs/src/usage/eks-managed-nodes.md [hunk 1]
```diff
@@ -150,6 +150,12 @@ existing cluster:
 $ eksctl create nodegroup --managed
 ```
 
+Tip : if you are using only one *`YOUR_CLUSTER.yaml`* file to describe your whole cluster, describe your new managed node group in its `managedNodeGroups` directive and run\:
```
Comment (cPu1) [inline userdocs/src/usage/eks-managed-nodes.md:153]: I think this is more clear:

```suggestion
Tip : if you are using a `ClusterConfig` file to describe your whole cluster, describe your new managed node group in its `managedNodeGroups` field and run\:
```

File: userdocs/src/usage/eks-managed-nodes.md [hunk 2]
```diff
@@ -150,6 +150,12 @@ existing cluster:
...
+
+```console
+$ eksctl create nodegroup --config-file=YOUR_CLUSTER.yaml --include='YOUR_MANAGEDNODEGROUP_NAME'
```
Comment (cPu1) [inline userdocs/src/usage/eks-managed-nodes.md:156]: By default, `eksctl create nodegroup` detects non-existing nodegroups and creates them, so `--include` can be omitted. 

```suggestion
$ eksctl create nodegroup --config-file=YOUR_CLUSTER.yaml'
```

**Review by cPu1 — CHANGES_REQUESTED** [review]
Thanks for the PR! I left some minor comments but otherwise LGTM.

**Comment by lboix** [comment]
> Thanks for the PR! I left some minor comments but otherwise LGTM.

I saw them and validated

> Thanks for the PR! I left some minor comments but otherwise LGTM.

Thanks for your feedback! I saw them and applied them

File: userdocs/src/usage/eks-managed-nodes.md [hunk 3]
```diff
@@ -150,6 +150,12 @@ existing cluster:
 $ eksctl create nodegroup --managed
 ```
 
+Tip : if you are using a `ClusterConfig` file to describe your whole cluster, describe your new managed node group in its `managedNodeGroups` field and run\:
+
+```console
+$ eksctl create nodegroup --config-file=YOUR_CLUSTER.yaml'
```
Comment (cPu1) [inline userdocs/src/usage/eks-managed-nodes.md:156]:
```suggestion
$ eksctl create nodegroup --config-file=YOUR_CLUSTER.yaml
```

**Review by cPu1 — APPROVED** [review]

**Pull request merged**